pyproject.toml

```diff
@@ -5,15 +5,18 @@ description = "A simple ToDo app for your DevOps apprenticeship"
 authors = []
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.9"
 flask = "^2.0.0"
 python-dotenv = "^0.17.1"
 requests = "^2.24.0"
 gunicorn = "^20.1.0"
+pymongo = {version = "^3.11.4", extras = ["srv", "tls"]}
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.4"
 selenium = "^3.3.0"
+pylint = "^2.8.2"
```

Comment:
Nice - if you're expecting developers to use this as part of their standard workflow then it would be great to get it hooked up to your integration pipeline too (but definitely a stretch goal, so feel free to ignore!)

Reply:
I have been working on this but it will be in a separate PR